README.md

```diff
@@ -1,2 +1,3 @@
 # hello-world
 Make things less complicated
+data caputer, exraction from files with common attributes
```

Comment:
yes.very helpful.would like to add a time stamp at time of data capture along with source address.

Reply:
helpful